src/components/menu/examples/menu-basic-secondary-text.tsx

```diff
@@ -0,0 +1,66 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * With `secondaryText`
+ *
+ * Menu items can display secondary text as well. By default, the secondary text
+ * will be displayed in two lines, and then get truncated.
+ *
+ * :::important
+ * Keep in mind that menu lists stretch as much as they can to display
```

Comment:
```suggestion
 * Keep in mind that menus stretch as much as they can to display
```

"menu list" doesn't really make sense to a user of Lime Elements.

Reply:
Why not? It's the list that opens when you press a menu. Something like dropdown list. 
Here I wasn't referring to the menu-list component actually. I was referring to the list of menu 😄

Reply:
Yeah, I understand. But that _is_ the menu. You "open a menu" by clicking on its button, you don't "open a menu list" by clicking its menu. I've never in my life heard anyone refer to the dropdown as a "menu list". If you want to specify that it's the dropdown you mean, not the button, then I'd call it that: "menu dropdown"

Reply:
```suggestion
 * Keep in mind that a menu's drop-down will stretch as much as it can to display
```